modules/command/src/main/java/org/apache/fluo/command/FluoOracle.java

```diff
@@ -30,8 +31,8 @@
   private static final Logger log = LoggerFactory.getLogger(FluoOracle.class);
 
   public static void main(String[] args) {
-    if (args.length != 2) {
-      System.err.println("Usage: FluoOracle <connectionPropsPath> <applicationName>");
+    if (args.length < 2) {
+      System.err.println("Usage: FluoOracle <connectionPropsPath> <applicationName> args...");
```

Comment:
instead of `args...` could this call call the JCommander help generator or should we just put the -D arg here?